flink-examples/flink-examples-table/src/main/java/org/apache/flink/table/examples/java/StreamWindowSQLExample.java

```diff
@@ -30,6 +30,8 @@
 /**
  * Simple example for demonstrating the use of SQL in Java.
  *
+ * <p>Usage: <code>./bin/flink run ./examples/table/StreamWindowSQLExample.java</code></p>
```

Comment:
use `{@code }` instead

the `</p>` is not important

JavaDocs are just pseudo HTML.